### PR TITLE
WIP - Add documentation for Background component

### PR DESCRIPTION
**Pull request by maddles**
This PR: 

* Updates the Victory version in package.json to `34.3.0`
* Adds documentation for the new `backgroundComponent` prop for `VictoryChart`
* Updates the documentation for the `style` prop to include `background` for `VictoryChart`
* Adds documentation for the new `Background` primitive component
* Adds to the `Introduction` a new section called `New Features` with documentation about the new features for `VictoryChart`

### CODE REVIEW

File: src/content/docs/victory-chart.md [hunk 1]
```diff
@@ -45,6 +45,18 @@ animate={{
...
 
+## backgroundComponent
+
+`type: element`
+
+The `backgroundComponent` prop takes a component instance which will be responsible for rendering a background if the `VictoryChart`'s `style` component includes `background` styles. The new element created from the passed `backgroundComponent` will be provided with the following properties calculated by `VictoryChart`: `height`, `polar`, `scale`, `style`, `x`, `y`, `width`. All of these props on `Background` should take prececence over what `VictoryChart` is trying to set.
+
+_default:_ `<Background/>`
+
+```jsx
```
Comment (boygirl) [inline src/content/docs/victory-chart.md:56]: It would be great to have a playground example here instead

Comment (wsparsons) [reply src/content/docs/victory-chart.md:56]: ok i will add an example

File: src/content/docs/victory-primitives.md [hunk 2]
```diff
@@ -149,6 +149,28 @@ const TSpan = (props) => <tspan {...props} />;
...
+  - `height` *number* the height of the `<rect/>` element
+  - `id` *string or number* an id to apply to the rendered component
+  - `origin` *object* the svg coordinates of the center point of a polar chart
+  - `polar` *boolean* a flag specifying whether the component is part of a polar chart
+  - `rectComponent` *element* the rendered rect element  *default* `<Rect/>`
+  - `role` *string* the aria role to assign to the element
+  - `scale` *object* the x and y scale of the parent chart with `domain` and `range` applied
+  - `shapeRendering` *string* the shape rendering attribute to apply to the rendered path
+  - `style` *object* the styles to apply to the rendered element
+  - `width` *number* the width of parent chart (used to calculate default bar width `style.width` is not supplied)
```
Comment (boygirl) [inline src/content/docs/victory-primitives.md:170]: This `width` prop is not correct

File: src/content/docs/victory-primitives.md [hunk 3]
```diff
@@ -149,6 +149,28 @@ const TSpan = (props) => <tspan {...props} />;
...
+  - `x` *number* the x coordinate of the top of the background
```
Comment (boygirl) [inline src/content/docs/victory-primitives.md:171]: top left for non polar charts, center for polar charts

File: src/content/introduction/new.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  - sampleData
+---
+
+# New Features
+
+An intro about new features! A link to the changelog
+
+## Backgrounds for VictoryChart
+
+VictoryChart now includes an optional [`backgroundComponent`](/docs/victory-chart#backgroundcomponent) prop. It will also render the new [`Background`](/docs/victory-primitives#backgroundcomponent) simple component as the default backgroundComponent when the "background" property is included in the VictoryChart style prop.
```
Comment (boygirl) [inline src/content/introduction/new.md]: I would love to see the copy on this page be a little more conversational. I can help out here if you'd like.

**Pull request merged**